modules/47_regipol/regiCarbonPrice/datainput.gms

```diff
@@ -109,9 +109,8 @@ loop(ext_regi$altFeEmiFac_regi(ext_regi),
 $endif.altFeEmiFac
 
 
-*** VRE capacity factor adjustments for Germany in line with ARIADNE assumptions
-$ifthen.GerVRECapFac not "%cm_ariadne_VRECapFac_adj%" == "off" 
-loop(te$sameas(te,"wind"),
+*** VRE capacity factor adjustments for Germany in line with results from detailed models in ARIADNE project
```

Comment:
I would suggest to remove this code from remind and instead add this change directly to the "calcCapacityFactor" function in mrremind, as the "f_cf.cs3r" input file exists to provide exactly that input to remind. 

If you open the function you will see that this is already done there for Germany coal capacity factors for example.

That way you also avoid the need to have this code repeated in the none realization, and its results will be also considered in a H12 run.

Reply:
I agree. Let's include that in some future clean-up when we scan the datainput files for stuff that can be moved to mrremind.